easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'preseq'
+version = '2.0.2'
+
+homepage = 'http://smithlabresearch.org/software/%(name)s'
+description = """Software for predicting library complexity and genome coverage in
+ high-throughput sequencing."""
```

Comment:
style nitpicking: this fits on a single line with < 120 chars

```python
description = "Software for predicting library complexity and genome coverage in high-throughput sequencing."

Reply:
@boegel I changed this.